### PR TITLE
[AutoScheduler] Add tips on resuming the search from a log file

### PR DESCRIPTION
**Pull request by merrymercy**
Auto-scheduler has very good support for resuming the search from a log file. We just need to add one more parameter when creating the task scheduler.
```python
tuner = auto_scheduler.TaskScheduler(tasks, task_weights, load_log_file=log_file)
```

### CODE REVIEW

**Review by masahi — APPROVED** [review]

File: python/tvm/auto_scheduler/task_scheduler.py [hunk 1]
```diff
@@ -94,8 +94,18 @@ def make_search_policies(
             raise ValueError("Invalid search policy: " + search_policy)
 
         if policy_type == "sketch":
+            if load_log_file:
```
Comment (comaniac) [inline python/tvm/auto_scheduler/task_scheduler.py:97]: Do we need to check if the log file exists?

**Review by comaniac — APPROVED** [review]

File: python/tvm/auto_scheduler/task_scheduler.py (hunk 1 above)
Comment (merrymercy) [reply python/tvm/auto_scheduler/task_scheduler.py:97]: No. This function correctly handles this case and prints
```
SearchPolicy: No measurement records found in XXX.json
```

**Comment by masahi** [comment]
A new flaky test from topi argwhere https://ci.tlcpack.ai/blue/organizations/jenkins/tvm/detail/PR-7039/1/pipeline

cc @zhiics

File: python/tvm/auto_scheduler/task_scheduler.py [hunk 2]
```diff
@@ -94,8 +94,18 @@ def make_search_policies(
...
+                init_search_callbacks = [PreloadMeasuredStates(load_log_file)]
```
Comment (merrymercy) [inline python/tvm/auto_scheduler/task_scheduler.py:98]:
```suggestion
                # use the log file to restore the status of search policies.
                init_search_callbacks = [PreloadMeasuredStates(load_log_file)]
```

**Pull request merged**